Mathlib/Algebra/BigOperators/Finsupp.lean

```diff
@@ -97,7 +97,7 @@ theorem sum_ite_self_eq [DecidableEq α] {N : Type*} [AddCommMonoid N] (f : α 
     (f.sum fun x v => ite (a = x) v 0) = f a := by
   classical
     convert f.sum_ite_eq a fun _ => id
-    simp [ite_eq_right_iff.2 Eq.symm]
+    simp [(ite_eq_right_iff _ _ _).2 Eq.symm]
```

Comment:
Apologies for these changes, they were a mistake and will be reverted back again...